scan

```diff
@@ -2,33 +2,76 @@
 
 import multiprocessing
 import os
+import uuid
 import sys
 import glob
-from scanners import utils
+import time
 import datetime
 import logging
 import requests
 import importlib
 import shutil
 import csv
+import json
+import base64
+import boto3
+import botocore
 from concurrent.futures import ThreadPoolExecutor
 
+from scanners import utils
+
 # basic setup - logs, output dirs
 options = utils.options()
 domain_suffix = options.get("suffix")
 utils.configure_logging(options)
 utils.mkdir_p(utils.cache_dir())
 utils.mkdir_p(utils.results_dir())
 
-# some metadata about the scan itself
-start_time = utils.utc_timestamp()
+# Default and maximum for local workers (threads) per-scanner.
+default_workers = 10
+global_max_workers = 1000
+
+# Some metadata about the scan itself.
+start_time = utils.local_now()
 start_command = str.join(" ", sys.argv)
 
+# Generate a random UUID for the entire scan.
+scan_uuid = str(uuid.uuid4())
```

Comment:
Not currently using this anywhere, but for ease of finding logs in Lambda, every scan gets a `uuid`. This is also made for local scans, and stored in `meta.json`.